cfgov/unprocessed/css/organisms/email-popup.less

```diff
@@ -13,10 +14,15 @@
     .respond-to-max( @bp-xs-max, {
         margin-left: -135px;
         left: 50%;
+
+        input[type="submit"] {
```

Comment:
Use `button`?

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/4838/commits/75324dfce4ced3b5116c004b5922521ac15ddf4e